src/InvidividualFileMessageWriter.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace ZwiftPacketMonitor
+{
+    public class InvidividualFileMessageWriter : IMessageWriter
```

Comment:
This is purely for debugging/analysing indeed. It allows you to have individual protobuf messages written to disk so that you can use deprotobuf or protoc to then inspect them. 

This was fairly helpful when figuring out which message types existed from the companion app as it saves you copying the byte stream directly from the debugger etc

Reply:
I've added a XMLDoc comment to it to explain what it's for